api/sexy.go

```diff
@@ -7,7 +7,7 @@ import (
 	"time"
 )
 
-func configGinSexyApiRouter(router gin.IRoutes, daoWrapper dao.DaoWrapper) {
+func configGinSexyApiRouter(router *gin.Engine, daoWrapper dao.DaoWrapper) {
```

Comment:
Can you remove the sexy ap (and also sadly the tests)? This was not used and the project that intended to use it is now archived.